packages/E2ETest/windows/ReactUWPTestApp/TreeDumpControlViewManager.cs

```diff
@@ -204,6 +204,10 @@ internal static async Task<bool> MatchTreeDumpFromLayoutUpdateAsync(string dumpI
             }
 
             string dumpText = VisualTreeDumper.DumpTree(dumpRoot, textBlock, additionalProperties, mode);
+
+            // #6944 VisualTreeDumper.DumpTree can add trailing commas
+            dumpText = Regex.Replace(dumpText, @"\},(\s+)\]", "}$1]", RegexOptions.Multiline);
+
```

Comment:
This is dirty but pretty harmless. Not wanting to wait on a change from another repo.

Reply:
Nvm, looks like it will be fixed before merge. WIll remove this before merging.